src/commands/index.ts

```diff
@@ -69,6 +72,14 @@ export function load(client: any): any {
   client.experimental = {};
   client.experimental.functions = {};
   client.experimental.functions.shell = loadCommand("experimental-functions-shell");
+  if (experiments.isEnabled("experiments")) {
```

Comment:
I assume this is eventually going to be removed before submitting? How would I get to these commands otherwise?

Reply:
Ah, I see previews still works. :)